xhttp/retriable/retriable.go

```diff
@@ -608,7 +618,7 @@ var nonRetriableErrors = []string{
 func (p *Policy) ShouldRetry(r *http.Request, resp *http.Response, err error, retries int) (bool, time.Duration, string) {
 	if err != nil {
 		errStr := err.Error()
-		logger.Errorf("api=ShouldRetry, error_type=%T, err=%q", err, errStr)
+		logger.Errorf("api=ShouldRetry, error_type=%T, err=[%s]", err, errStr)
```

Comment:
Not a big deal but in debugResponse %v is used for err.Error() and here you use %s. Should we be consistent?